packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -364,6 +368,8 @@ class ListTile extends StatelessWidget {
   /// If this property is null then its value is based on [ListTileTheme.dense].
   ///
   /// Dense list tiles default to a smaller height.
+  ///
+  /// When [ThemeData.useMaterial3] is true, [dense] wil not have any effect.
```

Comment:
This doesn't appear to be true. I think what we want to suggest is that we don't recommend that Material3 apps set dense to true.

Reply:
You're right. We're not skipping `dense` for M3.